### PR TITLE
Bcrypt Password Encryption

### PR DESCRIPTION
**Pull request by iammiracle**
What does this PR do?

- In this PR, the password is encrypted so that Auth0 login script can authenticate registered users.

### CODE REVIEW

**Review by olajohn-ajiboye — APPROVED** [review]
This is already well done. If you could  just remove the unnecessary white spaces.

**Comment by iammiracle** [comment]
> This is already well done. If you could just remove the unnecessary white spaces.

@olajohn-ajiboye I will appreciate if you can point to where I have white spaces. Thanks

File: server/controllers/userControllers.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
 /* eslint-disable camelcase */
 /* eslint-disable no-undef */
 /* eslint-disable sort-keys */
+/* eslint arrow-body-style: 0 */
+/* eslint func-names: 0 */
+/* eslint arrow-body-style: 0 */
+/* eslint prefer-arrow-callback: 0 */
+/* eslint consistent-return: 0 */
```
Comment (Way2nnadi) [inline server/controllers/userControllers.js]: remove all of these disabling of eslints

File: server/validation/user.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint arrow-body-style: 0 */
```
Comment (Way2nnadi) [inline server/validation/user.js]: remove eslint disable

**Review by enyeInc — APPROVED** [review]
This looks very good

**Pull request merged**